### DISCUSSION TITLE
Error using fatrop solver

### DISCUSSION THREAD

**Discussion by rtapia8484** [casadi/casadi#3827] [Q&A] [answered]
After executing the opti.solve(), fatrop completes its execution but I receive the following error 
![image](https://github.com/user-attachments/assets/3cc00d3d-1ae2-46a9-aaad-27e0ab3ed435)

Any clues as to why? Runs fine using ipopt solver.  (python on Windows)

**Comment by maximvochten** [comment, marked as answer]
Your issue is explained here: https://github.com/jgillis/fatrop_demo/issues/3

**Reply by rtapia8484** [reply]
Thank you.
